src/osd/OSD.cc

```diff
@@ -7330,7 +7330,7 @@ void OSD::ms_fast_dispatch(Message *m)
     tracepoint(osd, ms_fast_dispatch, reqid.name._type,
         reqid.name._num, reqid.tid, reqid.inc);
   }
-  op->osd_parent_span = tracing::osd::tracer.start_trace("op-request-created");
+  op->osd_parent_span = tracing::osd::tracer.add_span("op-request-created", m->otel_trace);
```

Comment:
please keep current behavior. if `m->otel_trace` is null, just start a new trace